### DISCUSSION TITLE
How to perform single point simulation of E3SM FATES on Perlmutter

### DISCUSSION THREAD

**Discussion by sugerWu-2822** [NGEET/fates#1209] [General]
Dear all,

I have research experience on porting the CESM1.0 and CLM4.5. Now I need to use the E3SM FATES model for single point simulation on the nersc serve Perlmutter. I wonder know if I need to successfully port E3SM to a new server first, and then turn on the FATES module for single point simulation testing?

Any suggestion are appreciated.

Sincerely,
Xiuyi Wu

**Comment by sshu88** [comment]
Hi Xiuyi,
    You might find the online tutorial to be helpful.
    https://fates-users-guide.readthedocs.io/en/latest/tutorials.html
    Since Perlmutter is a supported machine, I don't think any additional porting is required.

Best,
Shijie

**Reply by sugerWu-2822** [reply]
Hi Shijie,

Thank you for your reply. I have already read the online tutorial you mentioned about the FATES. 

You mentioned that Perlmutter doesn't need to be ported, so I'm wondering how I can run the E3SM-FATES model directly. I saw the part in the tutorial that taught us how to run a simple case, but it seems to be about CTSM (CESM), is E3SM similar? Is there a more detailed tutorial you know?

https://fates-users-guide.readthedocs.io/en/latest/running-fates-walkthrough-case1.html

Sincerely,
Xiuyi

**Reply by sshu88** [reply]
The tutorial material shall included certain script for running E3SM-FATES. On Perlmutter, all machine specs are recorded in corresponding XML files so when you create your case you can directly add argument "mach=pm-cpu". 
Hope these may help. I'm not pretty sure if another more detailed tutorial is available.

**Comment by sugerWu-2822** [comment]
Many thanks for you!

Xiuyi



---- Replied Message ----
| From | Shijie ***@***.***> |
| Date | 05/31/2024 08:54 |
| To | ***@***.***> |
| Cc | ***@***.***>***@***.***> |
| Subject | Re: [NGEET/fates] How to perform single point simulation of E3SM FATES on Perlmutter (Discussion #1209) |

The tutorial material shall included certain script for running E3SM-FATES. On Perlmutter, all machine specs are recorded in corresponding XML files so when you create your case you can directly add argument "mach=pm-cpu".
Hope these may help. I'm not pretty sure if another more detailed tutorial is available.
